src/main.rs

```diff
@@ -561,10 +561,7 @@ upload-addr = \"{}/{}\"
         self.invalidate_cloudfront(
             &self.config.cloudfront_static_id,
             &[
-                "/dist/channel*".into(),
-                "/dist/rust*".into(),
-                "/dist/index*".into(),
-                "/dist/".into(),
+                "/dist/*".into(),
```

Comment:
I still feel mildly uncomfortable just invalidating everything on every release, but I guess we don't release that often. It feels like at least we could only invalidate the date we pushed into (along with things in the root, I guess), but that is more work and it's unclear whether we gain much from keeping these caches around.

But no need to change anything here.

Reply:
Hmm, there are too many files on the top level to reliably invalidate, as we upload every tarball both on the date directory and the top level.